database_admin/schema/create_schema.sql

```diff
@@ -883,15 +883,20 @@ CREATE UNIQUE INDEX IF NOT EXISTS package_latest_cache_pkey ON package_latest_ca
 
 CREATE TABLE IF NOT EXISTS system_package
 (
-    rh_account_id INT NOT NULL REFERENCES rh_account,
-    system_id     INT NOT NULL,
-    package_id    INT NOT NULL REFERENCES package,
+    rh_account_id INT                                  NOT NULL REFERENCES rh_account,
```

Comment:
I think that cleaner would be to list columns in table definition in the same order as used in index, so: rh_account_id, name_id, package_id, system_id. The same for definition of gorm struct.

Reply:
Adding a column adds it at the end of the list, We can't just reorder table columns.